src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs

```diff
@@ -52,6 +52,12 @@ public class CleanseLogMessageFixture
         [TestCase(@"OutputPath=/home/mySecret/Downloads")]
         [TestCase("Hardlinking episode file: /home/mySecret/Downloads to /media/abc.mkv")]
         [TestCase("Hardlink '/home/mySecret/Downloads/abs.mkv' to '/media/abc.mkv' failed.")]
+        // Qbit Announce URLs
```

Comment:
No need to refer to qbit specifically `// Announce URLs (passkeys)`

Also include magneturl test, you'll see why.